test/apiv2/20-containers.at

```diff
@@ -245,6 +249,7 @@ t GET 'containers/json?limit=0&all=1' 200 \
 
 t GET containers/json?limit=2 200 length=2
 
+
```

Comment:
Could you please remove this extra line.

Reply:
done